src/pages/Admin/Details/ValidationSchema.ts

```diff
@@ -156,10 +163,15 @@ export const roomCreateSchema = Yup.object().shape<RoomSchemaShape>({
     ),
   url: Yup.string().when("useUrl", {
     is: true,
-    then: Yup.string()
-      .required("Url is required!")
-      .min(3, ({ min }) => `Url must be at least ${min} characters`),
+    then: roomUrlSchema,
   }),
+  image_url: roomImageUrlSchema,
+});
+
+export const roomEditSchema = Yup.object().shape<RoomSchemaShape>({
+  title: roomTitleSchema,
+  url: roomUrlSchema,
```

Comment:
In `roomCreateSchema`, this is only present within the `.when("useUrl")`. Is that logic also required here?

Reply:
It is not because that field is being used only to determine if it should validate url or venue name, when editing a room you are always validating the url.

Reply:
Cool. That works for me!